LongWarOfTheChosen/Src/LW_Overhaul/Classes/XComGameState_LWListenerManager.uc

```diff
@@ -401,66 +407,6 @@ function EventListenerReturn AddSquadSelectStripWeaponsButton (Object EventData,
 	return ELR_NoInterrupt;
 }
 
-function EventListenerReturn AddArmoryStripWeaponsButton (Object EventData, Object EventSource, XComGameState GameState, Name InEventID, Object CallbackData)
```

Comment:
Please don't remove these functions if they are still referenced by commented-out code in `InitListeners()`. I never finished the migration away from `XCGS_LWListenerManager`. What's left may still need to be migrated/implemented.